src/components/bluebutton.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './bluebutton.scss';
+
+const Button = ({ targetTo, label, ...props }) =>
+(
+  <div
+    className={styles.our_button}
+    role="button"
+    tabIndex="-1"
+    {...props}
+    onClick={(event) => {
+      event.preventDefault();
+      targetTo();
+    }}
+  >
+    {label}
+  </div>
+  );
+
+Button.propTypes = {
+  targetTo: PropTypes.func.isRequired,
+  label: PropTypes.string.isRequired,
+};
+
+Button.defaultProps = {
+  label: 'Name me!',
```

Comment:
This is not a useful default, you also don't need a default because you made `label` a required property.